index.html

```diff
@@ -0,0 +1,66 @@
+!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <a href="#" class="logo">
+        <p>Kalkidan</p>
+      </a>
+      <div class="union" id="union">
+        <img src="images/Union.png" alt="union icon" />
+      </div>
+    </header>
+    <section class="headline">
+      <h1 class="intro primary-text">
+        I am Kalkidan<br />
+        Glad to see <br />
+        you!
+      </h1>
```

Comment:
- [ ]  You should use similar fonts with the one provided in the Figma design. 
-
Your design     |   Figma design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/43877689/177412879-c4173a50-6fa2-4d83-8034-17f8c57744b1.png)  |  ![image](https://user-images.githubusercontent.com/43877689/177412784-c43e103c-2486-41fb-a195-88a972f54dd2.png)